src/ypd/servlets/project_servlet.py

```diff
@@ -59,6 +59,17 @@ def favorite(self, project):
         return redirect(url_for('ProjectView:view', id=project.id,
                                 is_provided=Tests.is_provided_test(project)))
 
+    # pull data from HTML form
+    @route('/archive', methods=('GET', 'POST'))
```

Comment:
Why accept POST? You aren't using a form for this

Reply:
Removed methods as they weren't needed.